src/components/Home/Home.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import Slider from '../../components/Slider/Slider';
+import ArticlesCategories from '../../components/Categories/Categories';
+import Modal from '../../components/Modal/Modal';
+import Header from '../../components/Header';
+import Feedback from '../../components/Feedback/Feedback';
+import Link from 'gatsby-link'
+import AvocadoTestImage from '../../pages/images/avocado.png';
+import {TEST_ARTICLES} from "../../pages/index";
+
+export default class HomeComponent extends React.Component {
+	constructor(props) {
+		super(props);
+		TEST_ARTICLES.map(article => {
+			article.more = false;
+		});
+		this.state = {
+			articles: props.articles || [],
+			modalConfig: {
+				id: "#about",
+				title: "About Service",
+				additionals: "Content here"
+			}
+		}
+	}
+	componentDidMount() {
+		this.state.articles ? this.state.articles.map(news => {
```

Comment:
In fact, don't get the idea completely... It seems you'd like to use forEach instead of map... + you should use the setState method to change the state... As for now, everything should work without this piece of code